KEYS

```diff
@@ -480,3 +480,40 @@ IxR3jVTKye+UerEtN8yATW8CRIKO3IobUfLMDdPCLO7uzoW95cI35Y0l8JgK2NeU
 =E4W5
 -----END PGP PUBLIC KEY BLOCK-----
 
+pub   rsa2048 2018-02-07 [SC] [expires: 2020-02-07]
```

Comment:
Since you're not going to make a release, I don't think that your key should be in here.